src/adm/filter.rs

```diff
@@ -221,17 +221,17 @@ impl AdmFilter {
                 } else {
                     filter
                 };
-                if let Err(e) = self.check_advertiser(adv_filter, &mut tile, tags) {
+                if let Err(e) = self.check_advertiser(adv_filter, &mut tile, tags.clone()) {
```

Comment:
this won't help as the `check_` methods end up with cloned tags but the `self.report` call handling the error doesn't use it (using the orig).

I'm fine reverting it back for now and fixing it later, I think it's more of an annoyance at this point

Reply:
crap, yeah, what I get for being clever. I added the tag modifiers right before we do the error gen.